core/src/main/java/org/testcontainers/utility/RyukResourceReaper.java

```diff
@@ -31,7 +31,7 @@ class RyukResourceReaper extends ResourceReaper {
         .withConstantThroughput()
         .build();
 
-    private final AtomicBoolean started = new AtomicBoolean(false);
+    private static final AtomicBoolean started = new AtomicBoolean(false);
```

Comment:
This one was required to make the ResourceReaperTest succeed. I don't understand, why it fails or why it didn't fail before.